### PR TITLE
fix(image): add example image - INNO-1257

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

The image we were using as example has been removed. A new image has been set; not it doesn't rely on external site.

### CODE REVIEW

File: src/systems/ec/specs/components/card/demo/data--card.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
+const demoImage = require('@ecl/ec-resources-example/example-image.jpg');
```
Comment (yhuard) [inline src/systems/ec/specs/components/card/demo/data--card.js]: Specs are meant to be generic. You're importing a jpg in a JS file. It implies people will have to use webpack or another module bundle in order to load this file...

Can you move the "require logic" outside of this file? Only the path to the image should remain here. :)

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 944a60eb6d3c600500e061bec74d63a8651510fd

https://deploy-preview-884--europa-component-library.netlify.com

**Review by yhuard — CHANGES_REQUESTED** [review]

File: src/systems/ec/resources/example/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "private": true,
+  "name": "@ecl/ec-resources-example",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.0.0-alpha.1",
```
Comment (yhuard) [inline src/systems/ec/resources/example/package.json]: please update version number :)

edit: everywhere

**Review by yhuard — CHANGES_REQUESTED** [review]

**Comment by emeryro** [comment]
Ready for another round

**Review by yhuard — APPROVED** [review]

**Pull request merged**